hbp_knowledge/oxidative-stress/hybertson2011.bel

```diff
@@ -78,142 +78,195 @@ SET Citation = {"PubMed","22020111"}
 activators"
+SET Confidence = "High"
 SET MeSHAnatomy = "Astrocytes"
 a(CHEBI:"dimethyl fumarate") -> act(p(HGNC:NFE2L2))
 UNSET MeSHAnatomy
 SET CellLine = "C6 cell"
 a(CHEBI:"dimethyl fumarate") cnc act(p(HGNC:NFE2L2))
 UNSET CellLine
+UNSET Confidence
 
 SET Evidence = "Sirtuin 1 (SIRT1) was shown to decrease acetylation of Nrf2, as well as Nrf2-dependent transcription."
+SET Confidence = "High"
 p(HGNC:SIRT1) -| p(HGNC:NFE2L2,pmod(Ac))
+UNSET Confidence
 
 SET Evidence = "In our gene expression data, SIRT1 was induced 1.75-fold by Protandim (p = 0.015)."
+SET Confidence = "High"
 a(PUBCHEM:135316034) -> p(HGNC:SIRT1)
+UNSET Confidence
 
 SET Evidence = "The study also found that resveratrol, a putative activator of SIRT1
 (Howitz et al., 2003), inhibited Nrf2-dependent transcription, apparently contradicting earlier reports that resveratrol activates
 Nrf2 (Chen et al., 2005; Ungvari et al., 2010)."
+SET Confidence = "High"
 a(CHEBI:resveratrol) -> act(p(HGNC:SIRT1))
 a(CHEBI:resveratrol) -| act(p(HGNC:NFE2L2))
+UNSET Confidence
 
 SET Evidence = "SLC7A11, induced 76-fold, encodes a cystine/glutamate antiporter responsible for maintaining
 extracellular glutamate in the brain, and for importing cystine necessary for glutathione synthesis (Albrecht et al., 2010)."
 # upregulation of gene expression by Protandim
+SET Confidence = "High"
 a(PUBCHEM:135316034) -> p(HGNC:SLC7A11)
 SET MeSHAnatomy = "Brain"
 p(HGNC:SLC7A11) reg bp(GO:"glutamate homeostasis")
 p(HGNC:SLC7A11) -> tloc(a(CHEBI:cystine),fromLoc(GO:"extracellular region"),toLoc(GO:intracellular))
 a(CHEBI:cystine) -> a(CHEBI:glutathione)
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "This antiporter was recently found to be decreased by repeated cocaine exposure, and restoration of the activity prevented
 cocaine-primed drug seeking behavior in rats (Baker et al., 2003)."
 #this antiporter: SLC7A11
+SET Confidence = "High"
 a(CHEBI:cocaine) -| act(p(HGNC:SLC7A11))
+UNSET Confidence
 
 SET Evidence = "AKR1B10, induced 72-fold, encodes aldo–keto reductase
 family 1 member B10 which efficiently detoxifies mutagenic and carcinogenic alpha, beta-unsaturated carbonyls such as
 4-hydroxynonenal (Zhong et al., 2009)."
 # upregulation of gene expression by Protandim
+SET Confidence = "High"
 a(PUBCHEM:135316034) -> p(HGNC:AKR1B10)
 p(HGNC:AKR1B10) -> bp(GO:detoxification)
 p(HGNC:AKR1B10) -| a(PUBCHEM:5283344)
+UNSET Confidence
 
 SET Evidence = "PTGR1 (aka LTB4DH), induced 68-fold, encodes leukotriene B4-12-hydroxydehydrogenase,
 which is considered to be a key enzyme responsible for biological inactivation of prostaglandins and related eicosanoids (Tai et al., 2002)."
 # upregulation of gene expression by Protandim
-a(PUBCHEM:135316034) -> p(HGNC:PTGR1)
-p(HGNC:PTGR1) -| act(a(MESH:Prostaglandins))
-p(HGNC:PTGR1) -| act(a(MESH:Eicosanoids))
+SET Confidence = "Medium"
+a(PUBCHEM:135316034) -> g(HGNC:PTGR1)
+g(HGNC:PTGR1) -| act(a(MESH:Prostaglandins))
```

Comment:
biological inactivation of prostaglandins is done by the PTGR1 protein and not by the gene
please check the whole document for this kind of issue